build/webpack.base.config.js

```diff
@@ -56,17 +56,16 @@ module.exports = {
     maxEntrypointSize: 300000,
     hints: isProd ? 'warning' : false
   },
+  mode: process.env.NODE_ENV || 'production',
   plugins: isProd
     ? [
-        new webpack.optimize.UglifyJsPlugin({
-          compress: { warnings: false }
-        }),
         new webpack.optimize.ModuleConcatenationPlugin(),
         new ExtractTextPlugin({
-          filename: 'common.[chunkhash].css'
+          filename: 'common.[chunkhash].css',
+          allChunks: true
```

Comment:
Why change it?

Reply:
https://github.com/webpack/webpack/issues/959

Reply:
"extract-text-webpack-plugin" get error : TypeError: Cannot read property 'call' of undefined
Fixing it, by adding allChunks: true
https://github.com/webpack/webpack/issues/959

Reply:
@RezaHaidari I can't see any issues with existing ExtractTextPlugin. Also following config isn't needed as well I guess as by setting mode to production it minifies the code anyway.

```
if (isProd) {
  config.optimization = {
    ...config.optimization,
    minimizer: [
      new UglifyJsPlugin({
        sourceMap: true,
        uglifyOptions: {
          compress: {
            inline: false,
          },
        },
      }),
    ],
  };
}
```

Any comments?

Reply:
https://github.com/vuejs/vue-hackernews-2.0/pull/293/commits/b4b2e334f17f8580c54488519111f3e3f3171559